sites/pycharm-guide/contents/tutorials/intro-aws/background/index.md

```diff
@@ -0,0 +1,99 @@
+---
+type: TutorialStep
+date: 2021-01-29
+title: Background on Serverless
+technologies: [AWS Toolkit]
+topics: [aws]
+author: mm
+subtitle: Introduce Microservices, AWS SAM (Serverless Application Model)
+thumbnail: ../thumbnail.png
+longVideo:
+  poster: ./poster_long.png
+  url: https://youtu.be/caW7eLThBwQ
+---
+
+Hello everyone, Welcome to the part one of the AWS PyCharm tutorial Series. 
+Today I will be explaining in brief about micro-services architectures and different serverless frameworks.
+
+
+# Background
+
+Before taking a deep dive into microservices, let's understand first 
+what is a monolithic architecture. In a traditional monolithic architecture,
+all processes are tightly coupled and run in a single service. 
+This means that if one process of the application experiences a spike in demand,
+then the entire architecture must be scaled. If any single service is not 
+functioning properly, then it's going to affect all the services.
+
+![microservices](./microservices.png)
+Image Source : Amazon Web Services (AWS)
+
+The scenario is completely different when it comes to microservices. 
+In a microservice architecture, the services are: loosely coupled, managed by
+a small team, easily scalable, and the most important part if one the services
+goes down, then it won't affect the other services. If one of your services
+is built using Java then your other services can also be developed, 
+but using python, .net or ruby. Microservices Architectures don't follow a one
+size fits all approach and provide flexibility to the developers, 
+as there is no technology constrained.
+
+There are lots of benefits when it comes to microservices.
+
+To know more about microservices I would recommend you to follow the link [aws.amazon.com/microservices](https://aws.amazon.com/microservices/)
+
+# AWS Lambda
+
+![aws_lambda](./aws_lambda.png)
+
+So in this course we will be focusing on one of the products that AWS offers
```

Comment:
As a suggestion, let's leave out "So" as the starting word of a sentence, except in certain cases. E.g. in the middle of a paragraph.